src/main/java/org/folio/consortia/service/FolioTenantService.java

```diff
@@ -39,6 +60,13 @@ protected void afterTenantUpdate(TenantAttributes tenantAttributes) {
     try {
       contextHelper.registerTenant();
       kafkaService.createKafkaTopics();
+      runInFolioContext(contextHelper.getSystemUserFolioExecutionContext(folioExecutionContext.getTenantId()), () -> {
+        if (isCustomFieldExist()) {
+          log.info("Custom-field already available with name {}", ORIGINAL_TENANT_ID);
```

Comment:
please add method names to log messages

Reply:
I applied at this place but at other places not sure if method call would be fit.

Reply:
It fits in all places. Can be skipped in log.error because it logs method name anyway

Reply:
fixed , thanks.